tests/test_stores.py

```diff
@@ -159,6 +160,9 @@ async def test_expires_in(store: Store) -> None:
     await store.set("foo", "bar", expires_in=10)
     assert math.ceil(await store.expires_in("foo") / 10) * 10 == 10  # type: ignore[operator]
 
+    frozen_datetime.tick(12)
+    assert await store.expires_in("foo") == -2
```

Comment:
So the reason this fails is because of a different bug. The redis backend is actually the only one behaving correctly here. 

If `expires_in` is called on an item that does not exist, it should return `None`. Since the item already expired here, Redis won't give us its TTL, since it was removed from memory.

The `expires_in` implementations of the other stores don't not go through the regular `get` method, which would not return an already expired item.

I suggest you xfail this for the other stores, since the test is actually correct, and we fix this in a different PR.